packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -160,7 +162,26 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
             return;
           }
 
-          // TODO verify that all found deep equal
+          let lastEvidence;
+          for (const store of found) {
+            const next = store.get(txHash);
+            if (lastEvidence) {
+              if (keyEQ(lastEvidence, next)) {
+                lastEvidence = next;
+              } else {
+                trace(
+                  '🚨 conflicting evidence for',
```

Comment:
is this a red-alert situation? It seems like we recover: we don't act on inconsistent evidence.

Reply:
I think it deserves immediate operational attention if two oracles are disagreeing. At least one is unreliable and may require immediate action.